publish/vault.go

```diff
@@ -65,6 +66,17 @@ func (vaultd *VaultDestination) UploadUnencrypted(data map[string]interface{}, f
 		}
 	}
 
+	existingSecret, err := client.Logical().Read(vaultd.secretsPath(fileName))
+	if err != nil {
+		return err
```

Comment:
I think it might be good to keep going regardless (after printing a warning) if there's an error reading the secret. I'm imagining a situation where someone has given SOPS permission to only _write_ to Vault.

Reply:
Good idea. Fixed this behavior in 01b5fb627997bfc3956c2a0ac3fec85c63ac5f76: log warn when no read access, log info when data not changed